extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -392,96 +417,95 @@ public function getData( $master = false ) {
 
-			$parts = self::explode( $title->getDBkey() );
+		$isStaff = (int)in_array( 'staff', $this->mUser->getEffectiveGroups() );
 
-			$buttons = []; // action links with full markup (used in Oasis)
-			$links = []; // action links with only a URL
-			$replyButton = '';
+		$parts = self::explode( $title->getDBkey() );
 
-			// this is for blogs we want to know if commenting on it is enabled
-			// we cannot check it using $title->getBaseText, as this returns main namespace title
-			// the subjectpage for $parts title is something like 'User blog comment:SomeUser/BlogTitle' which is fine
-			$articleTitle = Title::makeTitle( MWNamespace::getSubject( $this->mNamespace ), $parts['title'] );
-			$commentingAllowed = ArticleComment::canComment( $articleTitle );
+		$buttons = []; // action links with full markup (used in Oasis)
+		$links = []; // action links with only a URL
+		$replyButton = '';
 
-			if ( ( count( $parts['partsStripped'] ) == 1 ) && $commentingAllowed ) {
-				$replyButton = '<button type="button" class="article-comm-reply wikia-button secondary actionButton">' . wfMsg( 'article-comments-reply' ) . '</button>';
-			}
-			if ( defined( 'NS_QUESTION_TALK' ) && ( $title->getNamespace() == NS_QUESTION_TALK ) ) {
-				$replyButton = '';
-			}
+		// this is for blogs we want to know if commenting on it is enabled
+		// we cannot check it using $title->getBaseText, as this returns main namespace title
+		// the subjectpage for $parts title is something like 'User blog comment:SomeUser/BlogTitle' which is fine
+		$articleTitle = Title::makeTitle( MWNamespace::getSubject( $this->mNamespace ), $parts['title'] );
+		$commentingAllowed = ArticleComment::canComment( $articleTitle );
 
-			if ( $canDelete ) {
-				$img = '<img class="remove sprite" alt="" src="' . $wgBlankImgUrl . '" width="16" height="16" />';
-				$buttons[] = $img . '<a href="' . $title->getLocalUrl( 'redirect=no&action=delete' ) . '" class="article-comm-delete">' . wfMsg( 'article-comments-delete' ) . '</a>';
+		if ( ( count( $parts['partsStripped'] ) == 1 ) && $commentingAllowed ) {
+			$replyButton = '<button type="button" class="article-comm-reply wikia-button secondary actionButton">' . wfMsg( 'article-comments-reply' ) . '</button>';
+		}
+		if ( defined( 'NS_QUESTION_TALK' ) && ( $title->getNamespace() == NS_QUESTION_TALK ) ) {
+			$replyButton = '';
+		}
 
-				$links['delete'] = $title->getLocalUrl( 'redirect=no&action=delete' );
-			}
+		if ( $canDelete ) {
+			$img = '<img class="remove sprite" alt="" src="' . $wgBlankImgUrl . '" width="16" height="16" />';
+			$buttons[] = $img . '<a href="' . $title->getLocalUrl( 'redirect=no&action=delete' ) . '" class="article-comm-delete">' . wfMsg( 'article-comments-delete' ) . '</a>';
 
-			// due to slave lag canEdit() can return false negative - we are hiding it by CSS and force showing by JS
-			if ( $wgUser->isLoggedIn() && $commentingAllowed ) {
-				$display = $this->canEdit() ? 'test=' : ' style="display:none"';
```

Comment:
No clue, but the HTML for when a user could edit would be:
```html
<span class='edit-link'test=><img class=... /><a href=...</span>
```
Which is definitely broken HTML.  My guess is that the browser was just ignoring that extra text.